zipline/data/loader.py

```diff
@@ -31,10 +31,8 @@
 from . import benchmarks
 from . benchmarks import get_benchmark_returns
 
-from zipline.utils.tradingcalendar import (
-    trading_day,
-    trading_days
-)
+from zipline.utils.tradingcalendar import trading_day as trading_day_nyse
+from zipline.utils.tradingcalendar import trading_days as trading_days_nyse
```

Comment:
I decided to append "nyse" to the names to make their usage clearer. Tbh I feel tradingcalendar should also be named tradingcalendar_nyse.